### DISCUSSION TITLE
Show Tooltip after some delay

### DISCUSSION THREAD

**Discussion by Ri-gers** [DominoKit/domino-ui#978] [Q&A] [answered]
I'm wondering whether there is currently a way to show a Tooltip (currently using domino-ui 2.0.3) after some delay like browsers do with their default tooltips. The default domino Tooltip class appears directly on hovering which might be a bit too aggressive in some cases.

**Comment by vegegoku** [comment, marked as answer]
Currently there is not, but if you file an issue we can implement it in the next release.
